docs/content/how-to/cdc-ingestion.md

```diff
@@ -450,7 +450,7 @@ Synchronization from multiple Kafka topics to Paimon database.
 ### Prepare MongoDB Bundled Jar
 
 ```
-flink-sql-connector-mongodb-*.jar
+flink-sql-connector-mongodb-cdc-2.4.1.jar
```

Comment:
Document only cdc 2.4+ is supported.

Reply:
> Document only cdc 2.4+ is supported.

Should it be changed to `flink-sql-connector-mongodb-cdc-2.4.*.jar` like this?

Reply:
It is ok to keep this as it is.
I mean you can add another word to explain "only cdc 2.4+ is supported"

Reply:
done.